assets/apps/customizer-controls/src/global-colors/PaletteSelector.js

```diff
@@ -75,12 +75,16 @@ const PaletteSelector = ({ values, save }) => {
 										isDismissible={false}
 										className="neve-global-colors-confirm-delete-modal"
 										title={sprintf(
-											// translators: %s - name of palette that will be deleted.
+											// translators: %s: Context which will be deleted.
 											__(
-												'Are you sure you want to delete the %s palette?',
+												'Are you sure you want to delete the %s?',
 												'neve'
 											),
-											palettes[willDelete].name
+											sprintf(
+												'%1$s %2$s',
+												palettes[willDelete].name,
+												__('palette', 'neve')
+											)
```

Comment:
Let's keep the old string here, otherwise our translations % will go down because it's already translated. 😄

Reply:
@abaicus Nice catch, thank you 🚀 🚀  I've reverted just now!